frontend/src/pages/Users/Users.js

```diff
@@ -202,7 +209,8 @@ export default function Users() {
                 counter={ users && `(${Object.keys(users).length})` }
                 actions={
                   <SpaceBetween direction="horizontal" size="xs">
-                    <div onKeyPress={e => e.key == 'Enter' && createUser()}>
+                    {enableMfa && <Input onChange={({ detail }) => setState(userphonePath, detail.value)} value={userphone} placeholder='+11234567890'></Input>}
```

Comment:
Set `inputMode='tel'` this won't validate input but helps in certain cases for mobile, i.e. numeric keypad

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel

Reply:
https://github.com/aws-samples/pcluster-manager/pull/46/commits/ae3b86f7b6173a31fa412ba2030013c45f2aede5